components/elm/src/main/controlMod.F90

```diff
@@ -1123,7 +1124,13 @@ subroutine control_print ()
     else
         write(iulog,*) '   use_top_solar_rad is False, so do not run TOP solar radiation parameterization'
     end if
-    
+
+    if (use_finetop_rad) then
+        write(iulog,*) '  use fineTOP radiation parameterization instead of PP'
+    else
+        write(iulog,*) '   use_finetop_rad is False, so do not run fineTOP radiation parameterization'
+    end if
+
```

Comment:
Need to add a check that both `use_top_solar_rad` and `use_finetop_rad` can't be simultaneously true.

Reply:
Done.